src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -104,8 +105,15 @@ public void shootAtComputer(int row, int col) {
         }
         this.scanRequest = false;
     }
-
+    //splits based on AI level
     public void shootAtPlayer() {
```

Comment:
Hey, so in class we talked about how this kinda name isn't great because you are shooting at the player's board and not the player it's self.